internal/util/logging/console_handler.go

```diff
@@ -75,6 +92,23 @@ func (ch *consoleHandler) Enabled(_ context.Context, l slog.Level) bool {
 	return l >= minLevel
 }
 
+func coloredLevel(l slog.Level) string {
+	t := term.NewTerminal(nil, "")
+
+	switch l {
+	case slog.LevelInfo:
+		return string(t.Escape.Green) + l.String() + string(t.Escape.Reset)
```

Comment:
@AlekSi So, it's technically possible to use color escape codes from `term` package, but as they're [unexported](https://cs.opensource.google/go/x/term/+/master:terminal.go;l=26), the only way to get them is by creating Terminal, and then using its Escape field.

As we don't use term.Terminal (which takes io.ReadWriter, as an argument, and does things as taking an input, or autocomplete handling), initializing it on side just to takes those values doesn't make much sense. We can store necessary escape codes in variables on the logging package, and just use them as well.

The `term.IsTerminal` is still useful, as it will be used to decide whether colored output should be enabled.

Reply:
> which takes io.ReadWriter

So we use type assertion.

>  and does things as taking an input, or autocomplete handling

I believe it does that only in raw mode, no?

> We can store necessary escape codes in variables on the logging package

The whole point was to not have terminal magic outside of a dependency that deals with terminals.